mlflow/models/model.py

```diff
@@ -86,9 +106,7 @@ def from_dict(cls, model_dict):
         return cls(**model_dict)
 
     @classmethod
-    def log(cls, artifact_path, flavor, registered_model_name=None,
-            signature: ModelSignature = None, input_example: ModelInputExample = None,
```

Comment:
What's the motivation for removing `signature` and `input_examples` as explicit params to this method btw?

Reply:
They are not used by the log function anymore, instead they are passed to the flaver.save_model. So, if the flavor does not support signature / example, neither would Model.log_model.

Reply:
Got it makes sense! I'll actually open a follow-up PR to update this docstring to state a bit more clearly that `Model.log` calls the passed-in flavor's `save_model` & that [``kwargs``](https://github.com/mlflow/mlflow/pull/2775/files#diff-c470a0d352438df0c82bed9873047b3bR140) are passed to the flavor's `save_model` method.

Reply:
Cool, thx!